packages/binutils.rb

```diff
@@ -37,8 +25,10 @@ def self.build
               --enable-plugins \
               --disable-bootstrap \
               --enable-64-bit-bfd \
+              --enable-lto \
+              --enable-vtable-verify \
               --disable-werror"
-      system 'make'
+      system "make -j#{CREW_NPROC}"
```

Comment:
`-j#{CREW_NPROC}` is not necessary.  This is the default option.

Reply:
Noted.